File: cpp/Movie.cpp
#pragma once
#include <string>

using std::string;

class Movie
{
public:
	static const int CHILDRENS = 2;
	static const int NEW_RELEASE = 1;
	static const int REGULAR = 0;

	Movie(const string& title, int price_code)
		: title(title),
		priceCode(price_code)
	{
	}

	int getPriceCode()
	{
		return priceCode;
	}

	void setPriceCode(int price_code)
	{
		priceCode = price_code;
	}

	string getTitle()
	{
		return title;
	}

private:
	string title;
	int priceCode;
};

File: python/movie.py
class Movie:
    CHILDRENS = 2
    NEW_RELEASE = 1
    REGULAR = 0

    def __init__(self, title: str, price_code):
        self.__title = title
        self.__price_code = price_code

    def get_price_code(self):
        return self.__price_code

    def set_price_code(self, arg):
        self.__price_code = arg

    def get_title(self):
        return self.__title
